src/lib/registry/logger-registry.js

```diff
@@ -0,0 +1,58 @@
+const Logger = require('../logger');
+const EmptyRegistry = require('./empty-registry');
+const { throwIf } = require('../utils/type-checks');
+
+class LoggerRegistry extends EmptyRegistry {
+    constructor(notifierRegistry) {
+        super();
+        this.notifierRegistry = notifierRegistry;
+    }
+
+    add(name, notifierName) {
+        throwIf.not.string(name, 'name');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
        throwIf.not.string(name, "name");
```